examples/nextjs-lexical-ai/app/components/Loading.tsx

```diff
@@ -0,0 +1,7 @@
+export function Loading() {
+  return (
+    <div className="flex w-full h-full items-center justify-center flex-1 p-4">
+      <img src="https://liveblocks.io/loading.svg" alt="Loading" />
```

Comment:
The loading spinner should be gray and smaller, e.g. https://github.com/liveblocks/liveblocks/blob/main/examples/nextjs-lexical/app/loading.tsx#L7